src/pubsub/ua_pubsub_manager.h

```diff
@@ -22,6 +22,9 @@ typedef struct UA_PubSubManager{
     TAILQ_HEAD(UA_ListOfPubSubConnection, UA_PubSubConnection) connections;
     size_t publishedDataSetsSize;
     TAILQ_HEAD(UA_ListOfPublishedDataSet, UA_PublishedDataSet) publishedDataSets;
+#ifdef UA_ENABLE_PUBSUB_CONFIG
+    UA_String pubSubConfigFilename;
+#endif
```

Comment:
Instead of using internally a 'filename', we can store the encoded binary configuration directly in the pubsub-manager-structure. This makes portability easier. I think @jpfr also aims this behavior with his comment https://github.com/open62541/open62541/pull/3663#issuecomment-638472404. Of course, we can provide a function to load and write the config into a real file.